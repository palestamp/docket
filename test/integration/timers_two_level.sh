

rm -f ~/.docket_timer_test > /dev/null
set -v
docket timer new home
docket timer new work --abstract
docket timer new coding --parent work
docket timer new robocop --parent coding
docket timer new projects --parent home --abstract
docket timer new docket --parent projects
docket timer new hf --parent projects

docket timer set home event:onstart:stop work
docket timer set work event:onstart:stop home

docket timer start robocop
docket timer stat --status-only

docket timer start hf
docket timer stat --status-only

