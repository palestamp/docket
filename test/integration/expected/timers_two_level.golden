docket timer new home
docket timer new work --abstract
docket timer new coding --parent work
docket timer new robocop --parent coding
docket timer new projects --parent home --abstract
docket timer new docket --parent projects
docket timer new hf --parent projects

docket timer set home event:onstart:stop work
docket timer set work event:onstart:stop home

docket timer start robocop
docket timer stat --status-only
*               running   
home            stopped   
work            running   
coding          running   
robocop         running   
projects        stopped   
docket          stopped   
hf              stopped   

docket timer start hf
docket timer stat --status-only
*               running   
home            running   
work            stopped   
coding          stopped   
robocop         stopped   
projects        running   
docket          stopped   
hf              running   

